HunterPie.UI/Overlay/Widgets/Damage/View/MeterView.xaml

```diff
@@ -55,6 +56,20 @@
                                FontFamily="{StaticResource HUNTERPIE_DEFAULT_FONT}"
                                VerticalAlignment="Center"
                                FontSize="13"/>
+                    <TextBlock d:Text=" / "
+                               Text=" / "
+                               Visibility="{Binding QuestLife, Converter={StaticResource QuestLifeToVisibilityConverter}}"
+                               Foreground="{StaticResource HUNTERPIE_FOREGROUND_PRIMARY}"
+                               FontFamily="{StaticResource HUNTERPIE_DEFAULT_FONT}"
+                               VerticalAlignment="Center"
+                               FontSize="13"/>
+                    <TextBlock d:Text="3"
+                               Text="{Binding QuestLife, Mode=OneWay}"
+                               Visibility="{Binding QuestLife, Converter={StaticResource QuestLifeToVisibilityConverter}}"
+                               Foreground="{StaticResource HUNTERPIE_FOREGROUND_PRIMARY}"
+                               FontFamily="{StaticResource HUNTERPIE_DEFAULT_FONT}"
+                               VerticalAlignment="Center"
+                               FontSize="13"/>
```

Comment:
You're creating two extra UIElements here when you can just use one `TextBlock` with `Run` children, and instead of adding an extra `TextBlock`, just add the runs to the Death TextBlock element:
https://github.com/HunterPie/HunterPie/blob/67608749a5d0d455455dbd6f43258cecd13cd4f2/HunterPie.UI/Overlay/Widgets/Damage/View/MeterView.xaml#L53-L58

Here's an example of how to use `Run`:

https://github.com/HunterPie/HunterPie/blob/67608749a5d0d455455dbd6f43258cecd13cd4f2/HunterPie.UI/Overlay/Widgets/Damage/View/PlayerDamageView.xaml#L120-L138